### ISSUE TITLE
help text for variables only in one language possible

### ISSUE THREAD

**Issue by frase-git** [cmsimple-xh/morepagedata#6]
Oder habe ich da was übersehen?

**Comment by cmb69** [comment]
Ja, sieht so aus (genau wie für die Beschriftung). Nicht perfekt, aber damit sollte man doch eigentlich leben können, da man ja den einen Hilfstext auch mehrsprachig gestalten kann, oder?

**Comment by frase-git** [comment]
Natürlich kann man.
Schön ist es aber dennoch nicht.
Die config2.php (die ich sowieso eigenartig finde) müsste es für jede Sprache geben.

**Comment by cmb69** [comment]
> Die config2.php (die ich sowieso eigenartig finde) müsste es für jede Sprache geben.

Eigenartig finde ich config2.php ebenfalls; zumindest sollte die Datei eigentlich nicht von außen einsehbar sein, was aber so ist; ich würde sie im content/ Ordner mit der Erweiterung .json ablegen.

Allerdings würden sprachspezifische „Daten-Dateien“ auch wiederum ganz unterschiedliche Definitionen ermöglichen (andere Variablen mit anderen Werten). Das könnte wünschenswert sein, oder auch nicht. Falls nicht, dann könnte man mehrere Sprachen auch in der selben Datei speichern. Allerdings würde ich zuvor doch lieber das eigenartige JSON-Format überarbeiten. Ein Array von Objekten wäre wohl sinnvoller als das bisherige Objekt mit Array-Eigenschaften.

**Comment by frase-git** [comment]
Ja.

**Comment by cmb69** [comment]
[Es könnte so einfach sein …](#8)
